### ISSUE TITLE
`Error: Command failed: git ls-files ...` upon `vite` or `vite build`

### ISSUE THREAD

**Issue by magne4000** [vikejs/vike#1320]
### Description
Version: 0.4.148 (no issue with 0.4.147)

`git ls-files` command seems to be executed without a fallback for non git initialized project (which is the case of all Bati created repo, and its internal tests)

### Reproduction steps

```sh
pnpm create @batijs/app --solid --tailwindcss --eslint
cd my-app
pnpm i
pnpm run build
```

### Error log
```
react--express--trpc--eslint:build: $ vite build
react--express--trpc--eslint:build: error during build:
react--express--trpc--eslint:build: Error: Command failed: git ls-files "**/+*.js" "**/+*.ts" "**/+*.cjs" "**/+*.cts" "**/+*.mjs" "**/+*.mts" "**/+*.jsx" "**/+*.tsx" "**/+*.cjsx" "**/+*.ctsx" "**/+*.mjsx" "**/+*.mtsx" "**/+*.vue" "**/+*.svelte" "**/+*.marko" "**/+*.md" "**/+*.mdx" --exclude="**/node_modules/**" --exclude="dist/**" --exclude="**/*.telefunc.*" --others --cached --exclude-standard
react--express--trpc--eslint:build: fatal : ni ceci ni aucun de ses répertoires parents (jusqu'au point de montage /) n'est un dépôt git
react--express--trpc--eslint:build: Arrêt à la limite du système de fichiers (GIT_DISCOVERY_ACROSS_FILESYSTEM n'est pas défini).
react--express--trpc--eslint:build:
react--express--trpc--eslint:build:     at ChildProcess.exithandler (node:child_process:422:12)
react--express--trpc--eslint:build:     at ChildProcess.emit (node:events:517:28)
react--express--trpc--eslint:build:     at maybeClose (node:internal/child_process:1098:16)
react--express--trpc--eslint:build:     at Socket.<anonymous> (node:internal/child_process:450:11)
react--express--trpc--eslint:build:     at Socket.emit (node:events:517:28)
react--express--trpc--eslint:build:     at Pipe.<anonymous> (node:net:350:12)
react--express--trpc--eslint:build: error: script "build" exited with code 1 (SIGHUP)
react--express--trpc--eslint:build: ERROR: command finished with error: command (/tmp/bati/packages/react--express--trpc--eslint) bun run build exited (1)
```

**Comment by phonzammi** [comment]
This might refer to discussion [1313](https://github.com/vikejs/vike/discussions/1313)

**Comment by AurelienLourot** [comment]
It's a slightly different one actually. #1313 was about not having `git` installed. This one is about not being in a git repo, so one more case to be caught.

**Comment by brillout** [comment]
The problem is that the test `if ((err as Error).message.includes('not a git repository')) return null` doesn't match the French translation.

Either we use https://stackoverflow.com/questions/2180270/check-if-current-directory-is-a-git-repository/16925062#16925062, or we `return null` if any error occurs while running `$ git ls-files` with the risk of not getting notified if the `git ls-files` command fails for some other reason we should know about.

**Comment by brillout** [comment]
I'll release a new version tentatively tomorrow. Pre-released in `0.4.148-commit-70e7518` in the meantime.
